cfme/storage/volume_backup.py

```diff
@@ -0,0 +1,270 @@
+    title = Text('.//div[@id="center_div" or @id="main-content"]//h1')
+    flash = FlashMessages(
+        './/div[@id="flash_msg_div"]/div[@id="flash_text_div" or '
+        'contains(@class, "flash_text_div")]')
+
+    @property
+    def in_volume_backup(self):
+        return (
+            self.logged_in_as_current_user and
+            self.navigation.currently_selected == ['Storage', 'Block Storage', 'Volume Backups']
+        )
+
+    @property
+    def is_displayed(self):
+        return self.in_volume_backup
+
+
+class VolumeBackupAllView(VolumeBackupView):
+    """The all Volume Backup page"""
+    toolbar = View.nested(VolumeBackupToolbar)
+    including_entities = View.include(BaseEntitiesView, use_parent=True)
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_volume_backup and
+            self.title.text == 'Cloud Volume Backups')
+
+
+class VolumeBackupDetailsView(VolumeBackupView):
+    """The detail Volume Backup page"""
+    @property
+    def is_displayed(self):
+        expected_title = '{} (Summary)'.format(self.context['object'].name)
+
+        return (
+            self.in_volume_backup and
+            self.title.text == expected_title and
+            self.entities.breadcrumb.active_location == expected_title)
+
+    toolbar = View.nested(VolumeBackupDetailsToolbar)
+    sidebar = View.nested(VolumeBackupDetailSidebar)
+    entities = View.nested(VolumeBackupDetailsEntities)
+
+
+class VolumeRestoreView(VolumeBackupView):
+    """The restore Volume backup view"""
+    @property
+    def is_displayed(self):
+        return False
+
+    volume_name = BootstrapSelect(name='volume_id')
+    save = Button('Save')
+    reset = Button('Reset')
+    cancel = Button('cancel')
+
+
+@attr.s
+class VolumeBackup(BaseEntity, WidgetasticTaggable):
+    """ Model of an Storage Volume Backups in cfme
+
+    Args:
+        name: name of the backup
+        provider: provider
+    """
+    name = attr.ib()
+    provider = attr.ib()
+
+    def restore(self, name):
+        """Restore the volume backup. this feature included in 5.9 and above.
+
+        Args:
+            name: volume name
+        """
+        view = navigate_to(self, 'Restore')
+        view.volume_name.fill(name)
+        view.save.click()
+        view.flash.assert_success_message('Restoring Cloud Volume "{}"'.format(self.name))
+
+    def refresh(self):
+        self.provider.refresh_provider_relationships()
+        self.browser.refresh()
+
+    @property
+    def exists(self):
+        try:
+            navigate_to(self, 'Details')
+            return True
+        except BackupNotFoundError:
+            return False
+
+    @property
+    def size(self):
+        """ size of cloud volume backup.
+
+        Returns:
+            :py:class:`int' size of volume backup in GB.
+        """
+        view = navigate_to(self, 'Details')
+        return int(view.entities.properties.get_text_of('Size')[:-2])
```

Comment:
OPTIONAL: it should be accepted by int() by I suspect you forgot to remove space